myapp/app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<h1>Tasks</h1>
+
+<table class="table table-striped table-hover">
+  <thead class="table-dark">
+    <tr>
+      <th>Name</th>
+      <th>Desc</th>
+      <th>Status</th>
+      <th>Label</th>
+      <th>Priority</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @tasks.each do |task| %>
+      <tr>
+        <td><%= task.name %></td>
+        <td><%= task.desc %></td>
+        <td><%= task.status %></td>
+        <td><%= task.label %></td>
+        <td><%= task.priority %></td>
+        <td><%= link_to 'Show', task %></td>
+        <td><%= link_to 'Edit', edit_task_path(task) %></td>
+        <td><%= link_to 'Destroy', task, method: :delete, data: { confirm: 'Are you sure?' } %></td>
```

Comment:
it seems working on my env 

![Screen Shot 2021-06-17 at 12 41 49](https://user-images.githubusercontent.com/85204395/122327803-74060780-cf69-11eb-80ba-0c2066496b44.png)

Reply:
I do `bundle exec rails webpacker:install` in docker container then I can delete 😄 

if there are any steps required to set up a local environment, it would be good to put it in the description so that reviewers can easily check it 👍

Reply:
OK thank you i will add these steps on readme

Reply:
thanks! 😄

Reply:
I'd like you to run `rubocop` to your ruby files!

Reply:
OK 👍

Reply:
Done, corrected a lot of files

Reply:
Maybe you've run it on `tickets/step-8`, but I've confirmed it, so I'll accept this PR as well!

Reply:
yes, thank you